lld/COFF/Chunks.cpp

```diff
@@ -1045,13 +1108,9 @@ uint32_t MergeChunk::getOutputCharacteristics() const {
   return IMAGE_SCN_MEM_READ | IMAGE_SCN_CNT_INITIALIZED_DATA;
 }
 
-size_t MergeChunk::getSize() const {
-  return builder.getSize();
-}
+size_t MergeChunk::getSize() const { return builder.getSize(); }
```

Comment:
This seems like a poor formatting choice.

Reply:
I agree that this is worse.
`void setIsTagged` and `static unsigned getVerDefNum` in lld/ELF/Symbols.h are similar.

Reply:
There appears to be a `AllowShortFunctionsOnASingleLine` setting for clang-format that we can try setting: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortfunctionsonasingleline